internal/terminal/table.go

```diff
@@ -110,7 +110,7 @@ func (t table) dataString() string {
 				strings.Repeat(" ", t.columnWidths[header]-len(row[header])),
 			)
 		}
-		rows[i] = strings.Join(cells, gutter)
+		rows[i] = "  " + strings.Join(cells, gutter)
```

Comment:
this "two space" thing is used a lot of places (check out `gutter` in this file).

I think it'd be a good idea at this point to make `const gutter = "  " // gutter or spacer as a name?` and then use that in each place where appropriate.  (I also think we can get rid of that `strings.Repeat(" ", 2)` and just type out the string with two spaces)